### ISSUE TITLE
Update readthedocs.

### ISSUE THREAD

**Issue by ryneeverett** [ralphbean/bugwarrior#812]
The current commit is a few years old (5f296a82) and doesn't correspond with a release so maybe this is configured in the readthedocs or github UI?

**Comment by ryneeverett** [comment]
https://readthedocs.org/projects/bugwarrior/builds/ warns:

> The project bugwarrior-docs doesn't have a valid webhook set up, commits won't trigger new builds for this project. See the project integrations for more information.

**Comment by ryneeverett** [comment]
I created https://github.com/ryneeverett/bugwarrior-docs-mirroraction which is a repo that just contains a github actions workflow to run a daily cron job to update the mirror. Hopefully it works.

**Comment by wamserma** [comment]
The old documentation at https://bugwarrior.readthedocs.io is still up, which is a bit confusing for people looking up this project. Maybe get in contact with support@readthedocs.org to get this fixed.

**Comment by ryneeverett** [comment]
Thanks for the feedback @wamserma! I'm taking another shot at reaching out to other maintainers to see if we can recover the original readthedocs account first, but if that fails I will try to get it done by contacting the readthedocs team as you suggested.

**Comment by ryneeverett** [comment]
Hey @ralphbean, any chance you could help us out with one of the following:

1. fix the readthedocs build by enabling the webhook in this repository
2. redirect your bugwarrior.readthedocs.io to my up-to-date mirror bugwarrior-docs.readthedocs.io (not an ideal solution)
3. empower another maintainer with enough repository permission to enable the readthedocs webhook in this repository such as myself or @djmitche (this would be great regardless as your bus factor has been a bit of an issue)

**Comment by ralphbean** [comment]
@ryneeverett yes. I _think_ I just successfully enabled the webhook. Can you confirm?

I'm :+1: to empower you or @djmitche, but I don't think I see a way to do it while the repo is still under my namespace. WDYT about moving it to an org so you two can be granted full ownership rights?

**Comment by ryneeverett** [comment]
I was thinking about suggesting moving it to https://github.com/GothenburgBitFactory if they'd accept it.

**Comment by djmitche** [comment]
I'd be into that! Maybe @tbabej is the person to actually approve it?

**Comment by ryneeverett** [comment]
> I think I just successfully enabled the webhook. Can you confirm?

Next time we push to develop a new build should show up [here](https://readthedocs.org/projects/bugwarrior/builds/) or you could push the "Build Version" button by logging in to https://readthedocs.org/projects/bugwarrior.

However, the build will fail due to #1022. :upside_down_face:

**Comment by ryneeverett** [comment]
Well I pushed and it did not trigger a build.

**Comment by tbabej** [comment]
> I'd be into that! Maybe @tbabej is the person to actually approve it?

I'd say moving community projects under GBF umbrella is something we're generally open to, in the past we moved `tasklib` as an example. Especially when it comes to important community projects, like bugwarrior.

@ralphbean reading [the docs](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) looks like it might be easiest if you transfer `bugwarrior` under `tbabej/bugwarrior` and I then move it under `GothenburgBitFactory/bugwarrior`

**Comment by ralphbean** [comment]
Makes sense, @tbabej. Transfer request initiated.

**Comment by tbabej** [comment]
Both transfer completed - `bugwarrior` is now hosted at https://github.com/GothenburgBitFactory/bugwarrior

**Comment by ryneeverett** [comment]
How do we want to handle pypi credentials so we can cut releases?

**Comment by ryneeverett** [comment]
This issue is now resolved and bugwarrior.readthedocs.io is now building and displaying the latest docs!

Regarding the dismantling of bugwarrior-docs.readthedocs.io:

- I have deleted https://github.com/ryneeverett/bugwarrior-docs.
- I'm changing project links back to bugwarrior.readthedocs.io in #1026.
- I'm redirecting bugwarrior-docs.readthedocs.io to bugwarrior.readthedocs.io per [these instructions](https://docs.readthedocs.io/en/stable/user-defined-redirects.html#migrating-your-documentation-to-another-domain). This has not taken effect yet but I'm assuming that's just readthedocs slowness. If this doesn't work I will just delete my readthedocs account and that will delete the project which will soon be outdated.
